### DISCUSSION TITLE
No viable bin files were found

### DISCUSSION THREAD

**Discussion by c0urtneyt** [angelolab/toffy#303] [Q&A]
Hi,
I am running through the normalization, which has worked before, but for some reason no matter what file I use, it now comes up with an error saying "no viable bin files were found", even though the original run folder does have the bin files. 
<img width="820" alt="Bin" src="https://user-images.githubusercontent.com/40770583/206280071-294d30df-bb0e-4a9b-8e0b-330a91d3891d.PNG">
<img width="1426" alt="NoBinError" src="https://user-images.githubusercontent.com/40770583/206280073-afdbb0c6-3b27-4718-89ee-fe383689c51e.PNG">

Is there a way to correct this?

Thank you!

**Comment by ngreenwald** [comment]
@camisowers will you take a look

**Comment by camisowers** [comment]
Hi @c0urtneyt ! What that code is doing is reading the rosetta folder names with the pre-fix "fov" and checking for both .json and .bin files of the same name in the run directory. I don't think there have been any changes to the `write_mph_per_mass` function recently, so can you confirm that the rosetta FOV folder names haven't been changed and are still "fov-1-scan-1" and "fov-2-scan-1"? 

You can run the line `print(fovs)` to show what file names the function is trying to read in and let us know!
